src/elements/components/Lottie.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useRef } from 'react';
+import { dynamicImport } from '../../integrations/utils';
+
+let LOTTIE_PROMISE = Promise.resolve();
+
+const LOTTIE_URL =
+  'https://cdnjs.cloudflare.com/ajax/libs/lottie-web/5.9.6/lottie_light.min.js';
+
+export function loadLottieLight() {
+  LOTTIE_PROMISE = LOTTIE_PROMISE.then(() => dynamicImport(LOTTIE_URL));
+}
+
+export default function Lottie({ animationData }) {
+  const lottieRef = useRef();
+
+  useEffect(() => {
+    LOTTIE_PROMISE.then(() =>
+      global.lottie.loadAnimation({
+        container: lottieRef.current,
+        renderer: 'svg',
+        loop: true,
+        autoplay: true,
+        animationData
+      })
+    );
+  }, [lottieRef.current]);
```

Comment:
I'm pretty sure ref's don't work properly in a dep array because mutating the ref doesn't cause a re-render

That said, I'm assuming you've tested this so curious to get input from @drewj or @ondrosh

Reply:
hmm good point, it seems like recommended approach is to use state + callback instead. let me reafactor

Reply:
actually apparently you can just assume it's mounted in the useEffect, which makes sense